source/fuzz/fuzzer_util.h

```diff
@@ -497,6 +498,16 @@ opt::Instruction* GetLastInsertBeforeInstruction(opt::IRContext* ir_context,
                                                  uint32_t block_id,
                                                  SpvOp opcode);
 
+// Returns true if all of the members of OpTypeStruct instruction with result
+// id |struct_type_id| are decorated with BuiltIn decoration.
+// Otherwise (none of the members are BuiltIn), returns false.
```

Comment:
These options don't seem mutually exclusive.  The opposite of all members being decorated with BuiltIn is not that *none* are decorated with BuiltIn; it's that *at least one* is *not* decorated with BuiltIn.

From the way you have used this function it sounds like it should return true if and only if *at least one* member has BuiltIn.

If we are assuming the module is valid in the first place then as an integrity check we might want to assert that if any member has BuiltIn then all of them do.

I suppose the options are mutually exclusive in the context of a valid module - is that what you intended with the current comment?

Reply:
> I suppose the options are mutually exclusive in the context of a valid module - is that what you intended with the current comment?
Yes. According to the spec, if any struct member is builtin, all of its members should be builtin.

Reply:
Fixed in fdc3b5d.